players/g1_player.py

```diff
@@ -61,6 +139,7 @@ def f(choice: Choice) -> float:
             res += choice.max_depth * 0.2
             res += 0.01 * len(choice.flavors)
             return res
+
         choice = max(choices, key=f)
```

Comment:
remove duplicated function f
you may need `functools.partial`

Reply:
I will revise score function then, to try what you have said in wechat